src/iOS.Autofill/MainInterface.storyboard

```diff
@@ -1,7 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I see a lot of changes in this file unrelated to your new View Controller. Might be fine, but I don't know.

Reply:
Not sure about this note as of now. I wonder if it could be related to using VS generation/editing versus using Rider/Xcode generation/editing.